### PR TITLE
Update Image Links in oVirt's Quick Start Guide

### PR DESCRIPTION
**Pull request by leni1**
Fixes issue #879 

Changes proposed in this pull request:

- Update image references. Image names had been changed (in a lot of cases, the filename now began with a capital letter). Their formats were different as well (in the original, they were referenced as .jpg, but in the ovirt-site/source/images/wiki folder, most of the images are .png files)

I confirm that this pull request was submitted according to the [contribution guidelines](/CONTRIBUTING.md): @leni1

This pull request needs review by: @jasonbrooks

### CODE REVIEW

File: source/documentation/quickstart/quickstart-guide.html.md [hunk 1]
```diff
@@ -386,6 +386,7 @@ To connect to oVirt web management portal
 You have now successfully logged in to the oVirt web administration portal. Here, you can configure and manage all your virtual resources. The functions of the oVirt Engine graphical user interface are described in the following figure and list:
 
 ![Figure 1. Administration Portal Features](admin-portal-label.png "Figure 1. Administration Portal Features")
+ovirt-site/source/images/wiki/Admin-portal-label.png
```
Comment (jasonbrooks) [inline source/documentation/quickstart/quickstart-guide.html.md]: these need to be like this:

```
![Figure 1. Administration Portal Features](wiki/Admin-portal-label.png "Figure 1. Administration Portal Features")
```

Comment (leni1) [inline source/documentation/quickstart/quickstart-guide.html.md]: Oh. Great. Let me do that. Thanks for the review.

**Comment by leni1** [comment]
Added the references as requested by @jasonbrooks 
Waiting for review.

**Comment by jasonbrooks** [comment]
LGTM

**Comment by leni1** [comment]
Corrected image references as asked by @bproffitt 
Waiting for review.

**Comment by jasonbrooks** [comment]
Hmm, good to know -- it worked w/o the /images/ on the local dev server

**Pull request merged**